detox/start_android_emulator.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+set -o pipefail
+
+NAME=detox_pixel_4_xl_api_30
+
+# Create virtual device in a relative "detox_pixel_4_xl_api_30" folder
+avdmanager create avd -n $NAME -k 'system-images;android-30;google_apis;x86' -g google_apis -p $NAME -d 'pixel'
```

Comment:
Maybe you should check if the emu already exists?

Reply:
I see. I'll add the check for it.

Reply:
Done. If emulator exist, then it will return immediately with a message stating that it already exists.